.rubocop.yml

```diff
@@ -53,3 +53,9 @@ Style/MultilineBlockChain:
 # Allow Perl-style references to regex matches
 Style/PerlBackrefs:
   Enabled: false
+
+Style/Documentation:
+  Exclude:
+    - 'lib/reek/ast/sexp_extensions/send.rb'
+    - 'lib/reek/ast/sexp_extensions/super.rb'
+    - 'lib/reek/ast/sexp_extensions/variables.rb'
```

Comment:
And the award for the smallest possible nitpick goes to… please configure your editor to add newlines at the end of the files; POSIX says a text file consists of lines and line is characters + a newline. :tophat: :wink: